js/about/preferences.js

```diff
@@ -654,8 +654,13 @@ class SecurityTab extends ImmutableComponent {
 
 class AdvancedTab extends ImmutableComponent {
   render () {
-    const pocketL10nArgs = {
-      extensionName: 'Pocket'
+    const extensionL10nArgs = {
+      pocket: {
+        extensionName: 'Pocket'
+      },
+      honey: {
+        extensionName: 'Honey'
+      }
     }
```

Comment:
@bsclifton Looking at this a second time, should we instead go with something like this?

```js
['Pocket', 'Honey'].reduce( (curr, key) => {
    return (curr[key.toLowerCase()] = { extensionName: key }), curr
}, {})
```